pkg/server/validation/envvars.go

```diff
@@ -0,0 +1,29 @@
+package validation
+
+import (
+	"regexp"
+)
+
+var (
+	envVarNameRegexp = regexp.MustCompile("^[-._a-zA-Z][-._a-zA-Z0-9]*$")
```

Comment:
what about?
`^[-._a-zA-Z][-._\w]*$`

Reply:
Sure, I copied from k8s source code :see_no_evil:

Reply:
😂 it's a minor change, no problem for me